src/sentry/static/sentry/app/components/listView.jsx

```diff
@@ -66,9 +68,39 @@ class ListView extends React.Component {
     listActionBar: PropTypes.object,
   };
 
+  constructor() {
+    super();
+    this.state = {
+      expandedRows: [],
```

Comment:
This is something that Johan wrote (I overtook his branch), and I haven't reflected on this yet. The functionality for expanded rows is not implemented yet. But, since this will come soon, it's good to reflect on it now! Yes, it could be a good idea to move it to the redux state, although I have not settled my mind of the pros and cons of it yet. The guy that held the redux course at PluralSight claimed that you should not move state to redux unless it was necessary. He wasn't elaborate about it, but that made me curious of why.